data/json/npcs/talk_tags.json

```diff
@@ -410,7 +410,6 @@
       "Stay with me, <name_g>!",
       "Catch up, <name_g>!",
       "Keep up!",
-      "Come on, <catch_up>!",
```

Comment:
Perhaps it was intended to use `<keep_up>` here.

Reply:
Yep, but I decided to delete it instead because the capitalization would be wrong, e.g. `Come on, Keep up!!`